test/e2e/e2e_test.go

```diff
@@ -163,21 +161,21 @@ func TestE2EFlow(t *testing.T) {
 		assert.NoError(t, err, "NSTemplateSet id not deleted")
 
 		err = memberAwait.WaitUntilNamespaceDeleted(johnsmithName, "code")
-		assert.NoError(t, err, "johnsmith-code namnespace is not deleted")
+		assert.NoError(t, err, "johnsmith-code namespace is not deleted")
 
 		err = memberAwait.WaitUntilNamespaceDeleted(johnsmithName, "dev")
-		assert.NoError(t, err, "johnsmith-dev namnespace is not deleted")
+		assert.NoError(t, err, "johnsmith-dev namespace is not deleted")
 
 		err = memberAwait.WaitUntilNamespaceDeleted(johnsmithName, "stage")
-		assert.NoError(t, err, "johnsmith-stage namnespace is not deleted")
+		assert.NoError(t, err, "johnsmith-stage namespace is not deleted")
 
 		// also, verify that other user's resource are left intact
-		verifyResourcesProvisionedForSignup(t, awaitility, johnExtraSignup, revisions, "basic")
+		verifyResourcesProvisionedForSignup(t, awaitility, johnExtraSignup, "basic")
```

Comment:
We also need to add a check that the ClusterQuota is deleted too.

Reply:
it's already done - see 7b2eb76

Reply:
It wasn't when I was adding that comment :) Thanks for addressing it!